posthog/temporal/data_imports/pipelines/sql_database/__init__.py

```diff
@@ -76,6 +78,13 @@ def sql_source_for_type(
         credentials = ConnectionStringCredentials(
             f"mysql+pymysql://{user}:{password}@{host}:{port}/{database}?ssl_ca={ssl_ca}&ssl_verify_cert=false"
         )
+    elif source_type == ExternalDataSource.Type.PLANETSCALE:
```

Comment:
Can I be a massive pain here and suggest we don't add planetscale as a new source? I think it'd be a good idea to make generic sources and not platform-specific (we could end up going down quite a rabbit hole with that) - maintaining multiple versions of a MySQL connector isn't ideal.

We should be able to distinguish planetscale DBs from the hostname - any host ending in `psdb.cloud` is planetscale, we can add the connect args based on that

Reply:
good point, missed the bit with psdb!